### DISCUSSION TITLE
작은 bbox area annotation 제거

### DISCUSSION THREAD

**Discussion by jaehwan-AI** [boostcampaitech2/object-detection-level2-cv-06#14] [ideas]
- 사람이 알아보기 힘든 작은 bbox들을 제거해서 학습을 시도할 예정입니다.
- 작은 bbox의 기준은 다음과 같습니다.
  1. 작성자가 주관적으로 알아보기 힘든 크기 선별
  2. 선별된 bbox의 크기의 최대값을 기준으로 그 이하의 annotation을 제거

**Comment by ppskj178** [comment]
[참고 부탁드립니다]
https://github.com/boostcampaitech2/object-detection-level2-cv-06/discussions/11#discussioncomment-1412614
